packaging/windows/host/host.wxs

```diff
@@ -35,9 +34,13 @@
         </Directory>
     </Fragment>
     <Fragment>
-        <ComponentGroup Id="InstallFiles">
+        <ComponentGroup Id="InstallSharedHostandDetectionKeys">
```

Comment:
Both those scenarios work, i.e. the host remains on the box unless the last version of the product is uninstalled. 

There is a more complex scenario like below that will not be addressed by this:
1.Install with fix
2.Install 1.0.4
3.Install 1.0.2
4.Uninstall 1.0.4  -- Registry is busted, but host (dotnet.exe) remains.
5.Uninstall 1.0.5  -- host is removed busting 1.0.2. 

In step 5, Uninstalling 1.0.2 first instead of 1.0.5 does not remove the host.